src/mixins/_button.scss

```diff
@@ -89,6 +91,62 @@
     transform: none;
   }
 
+  .c-button__extra {
+    display: flex;
+    font-size: ms.step(1);
+    margin: 0 ms.step(-4);
+    position: relative;
+
+    &:first-child {
+      margin-left: ms.step(-4) * -1;
+    }
+
+    &:last-child {
+      margin-right: ms.step(-4) * -1;
+    }
+  }
+
+  /**
+   * Toggling button
+   * This adds a slash across the icon to visually show the state has changed
+   * when `aria-pressed` is `'true'`.
+   */
+  &[aria-pressed='true'] {
+    .c-button__extra {
+      &::after {
+        background: currentColor;
+        border-radius: size.$border-radius-full;
+        content: '';
+        height: size.$edge-medium;
+        left: 0;
+        position: absolute;
+        top: 50%;
+        transform: rotate(-30deg);
+        width: 100%;
+      }
+
+      $clip-path: polygon(
+        -5% -5%,
+        105% -5%,
+        105% 23%,
+        -5% 87%,
+        -5% 101.4711%,
+        105% 37.9626%,
+        105% 105%,
+        -5% 105%
+      );
+
+      & > * {
+        clip-path: $clip-path;
+        transform: translate(0, 0);
```

Comment:
@tylersticka The [prototype had `height: 100%` and `width: 100%`](https://github.com/cloudfour/cloudfour.com-patterns/pull/1302/files#diff-0c4e5bc28b5fd976b8b3719d6811a920ca3c6e930db1b7839220eb386e809d92R50-R52) but I removed them here because the icons would blow up when I zoomed the browser in or out. 😮 🤔 ¯\\\_(ツ)_/¯

![icons](https://user-images.githubusercontent.com/459757/121969430-0d92a680-cd29-11eb-9941-8d8060f3272e.gif)

Reply:
Whoa!! Neat! I wonder why?

Reply:
It was very interesting! Even more so that the icons wouldn't reset back to their original size when I zoomed back to 100%. 🤯